Fw/Types/Serializable.hpp

```diff
@@ -72,6 +80,17 @@ namespace Fw {
 
             SerializeStatus serialize(const U8* buff, NATIVE_UINT_TYPE length, bool noLength = false); //!< serialize data buffer
```

Comment:
Make old calls have DEPRECATED tag.

Reply:
Not going to do this.  To much would be forced to change....as deprecated is an error under `-Werror`.